signals_unix_test.go

```diff
@@ -0,0 +1,33 @@
+// +build aix darwin dragonfly freebsd linux netbsd openbsd solaris
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package containerd
+
+import (
+	"syscall"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestParseRtSignal(t *testing.T) {
+	sig, err := ParseSignal("SIGRTMIN+3")
```

Comment:
Can you please do more test cases here? including non-happy paths - "SIGRTMIN+", "SIGRTMIN+x", "SIGRTMIN+1b", etc).

Reply:
Looking what Docker supports closely, I'm now inclined to use moby/sys instead of implementing our own. Waiting https://github.com/moby/sys/pull/69 now. (cc @cpuguy83)